extensions/dicom-rt/src/index.js

```diff
@@ -51,8 +48,9 @@ export default {
               if (study && study.series) {
                 for (let j = 0; j < study.series.length; j++) {
                   const series = study.series[j];
-                  console.log(studies);
-                  if (['RTSTRUCT', 'RTPLAN', 'RTDOSE'].includes(series.Modality)) {
+                  if (
```

Comment:
I realise this was from a previous PR, but this only caught my eye just now. We should only show the side panel if we have an RTSTRUCT, for now. We don't have any support for the other two types, maybe just add a comment "Could be expanded to contain `RTPLAN` and `RTDOSE` information in the future".